isbnbot/find_editions_with_isbns.sh

```diff
@@ -0,0 +1,18 @@
+<<<<<<< HEAD
+#!/bin/bash
+
+if [[ -z $1 ]]
+  then
+    echo "No dump file provided"
+    exit 1
+fi
+if [[ -z $2 ]]
+  then
+    echo "No output file provided"
+    exit 1
+fi
+
+OL_DUMP=$1
+OUTPUT=$2
+
+zgrep ^/type/edition $OL_DUMP | grep -E '"isbn_(10|13)":' |  grep -v -E '"isbn_(10|13)": \["\d+"\]' | pv | gzip > $OUTPUTls
```

Comment:
I think there are some typos on this line; also let's make sure to check that the regex works; I think we might need `[0-9]` instead of `\d` (grep's regex syntax is a little weird).